spotify/widget.go

```diff
@@ -0,0 +1,90 @@
+package spotify
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gdamore/tcell"
+	"github.com/rivo/tview"
+	"github.com/senorprogrammer/wtf/wtf"
+	"github.com/sticreations/spotigopher/spotigopher"
+)
+
+const HelpText = `
+  Keyboard commands for Textfile:
+	TESTTEST
+	`
```

Comment:
This all looks great except for this part. Could you add descriptive help text for the keyboard commands, and then I'll merge to master?

Reply:
Oops :) 
I added a Helptext.

Reply:
Awesome, thank you!